.github/workflows/kind.yml

```diff
@@ -15,20 +15,13 @@ defaults:
     shell: bash
 
 jobs:
-  kind-tests:
+  quick-checks:
+    name: Quick Checks
     runs-on: ubuntu-latest
     env:
+      KIND_VERSION: latest
       REGISTRY: localhost:5000
-    strategy:
-      fail-fast: false
-      matrix:
-        # Run tests on minimum and newest supported OCP Kubernetes
-        # The "minimum" tag is set in the Makefile
-        # KinD tags: https://hub.docker.com/r/kindest/node/tags
-        kind:
-          - "minimum"
-          - "latest"
-    name: KinD tests
+    # no strategy.matrix; for simplicity this only uses a 'latest' KinD cluster
```

Comment:
(Optional) This feels more like a comment for the PR rather than a comment that lives in the code?

Reply:
Yeah, it was more helpful for me when writing these, to make sure they all had the same fields in the same order just to help readability. I'll remove it, and it will live on in this PR thread :laughing: